src/components/weather/DailyWeather.tsx

```diff
@@ -52,20 +52,24 @@ export const DailyWeather = ({ date, description, icon, temperatures }: Props) =
           <RegularText />
           <WrapperRow spaceBetween>
             <View style={styles.dayTimeEntry}>
-              <RegularText>Morgens</RegularText>
-              <RegularText>{morn.toFixed(1)}°C</RegularText>
+              <RegularText center>
+                Morgens {'\n'} {morn.toFixed(1)}°C
```

Comment:
do we need additional spaces for voice over here?

Reply:
We don't need extra spaces for VoiceOver here. I just added it to make the code look nicer. Unnecessary spaces can be deleted.